src/Collection/FilesCollection.php

```diff
@@ -245,4 +245,19 @@ public function ensureFiles(FilesCollection $files)
 
         return $newFiles;
     }
+
+    /**
+     * @return FilesCollection
+     */
+    public function ignoreSymlinks()
```

Comment:
Please use the `FilesCollection->filter()` method internally. These methods should be immutable instead of manipulating the internal array.